dotcom-rendering/src/web/lib/LiveBlogRenderer.tsx

```diff
@@ -59,9 +59,11 @@ export const LiveBlogRenderer = ({
 	availableTopics,
 	selectedTopics,
 }: Props) => {
+	const filtered = selectedTopics?.length || filterKeyEvents;
```

Comment:
I may be wrong but I think filtered could end up with a value of number or boolean. 
0 || true => true
0 || false => false
1 || false => 1
1 || true => 1

Would it be not better to make sure it's value is a boolean? maybe like this:

`const filtered = selectedTopics?.length > 0 || filterKeyEvents;`

Reply:
Sounds sensible! I'll swap this out

Reply:
After taking another look at this I've actually opted for a double bang here instead on selectedTopics as `[]` or `undefined` are falsey and so will return false when combined with `!!`. That way we don't need the additional length check.

Reply:
Maybe I'm doing it wrong but just did a test in js console and empty list seems to come as truthy 🤔 
![image](https://user-images.githubusercontent.com/15894063/178795617-af9719cd-685c-4732-9735-222c7a11283f.png)

Reply:
Ah, the joys of javascript. Great spot!

Reply:
You're totally right @marjisound, let me revert this!